packages/monorepo-scripts/src/publish.ts

```diff
@@ -84,7 +85,16 @@ async function confirmAsync(message: string): Promise<void> {
         await generateAndUploadDocJsonsAsync(packagesWithDocs, isStaging, shouldUploadDocs);
     }
     const isDryRun = configs.IS_LOCAL_PUBLISH;
-    await publishReleaseNotesAsync(updatedPublicPackages, isDryRun);
+    const releaseNotes = await publishReleaseNotesAsync(updatedPublicPackages, isDryRun);
+    utils.log('Published release notes');
+
+    if (!isDryRun && releaseNotes) {
+        try {
+            await alertDiscordAsync(releaseNotes);
+        } catch (e) {
+            utils.log("Couldn't alert discord, error: ", e.message, '. Please alert manually.');
```

Comment:
Let's add a note about how to publish still succeeded so people don't freak out at getting an error message.